src/components/Change/ChangeSummary.vue

```diff
@@ -71,7 +71,8 @@
 </template>
 
 <script lang="ts">
-import { Component, Prop, Vue } from 'vue-property-decorator'
+import Vue from 'vue'
+import { Component, Prop } from 'vue-property-decorator'
```

Comment:
I did this to reduce our dependency on vue-property-decorator package, since it might go away in Vue3.